### ISSUE TITLE
Decide which WCAG criterion to explore adding guidance for

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4682]
## What

Decide which WCAG criterion to explore adding more guidance for.
It should be something that can cause issues more frequently.

## Why

To be able to work on exploring potential guidance for that criterion

## Who needs to work on this

Accessibility specialist

## Who needs to review this

Squad

## Done when

- [x] Made the decision and shared it with the squad

**Comment by selfthinker** [comment]
Out of the twelve criteria from the monitoring and Craig's slides I half-randomly chose: 2.4.6 Headings and Labels.
